bundles/org.openhab.binding.pentair/src/test/resources/log4j.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
What's the purpose of these logging configuration files?

Reply:
When doing the junit testing, I was trying to figure out how to setup so I could see the debug messages.  Is there a documented way to do this?  The solution which worked for me was the simplelogger.properties file (and can remove the other 2 for sure).  I can remove the simplelogger.properties for the checkin in desired as well and just keep that local.

Reply:
Ok, would be better to remove all.